README.md

```diff
@@ -25,8 +25,6 @@ sudo apt-get install nodejs cmake build-essential swig python3.8
 sudo npm install --global npm
```

Comment:
I think the "repeat until" block is reversed. This runs forever:
![image](https://user-images.githubusercontent.com/126523/220707523-f9d9d86d-59f1-4710-8a1e-be0cd0194967.png)

while reversing the condition terminates immediately